ds-collector/collector.conf.in

```diff
@@ -84,6 +84,9 @@
 #
 #issueId="XXX-6"
 
+# The MD5 checksum of the provided .key file
+#keyMD5sum=
```

Comment:
generating this shouldn't be up to Reuben.

that should be done in this repo in the top-level https://github.com/datastax/diagnostic-collection/blob/master/Makefile